package.json

```diff
@@ -32,11 +32,14 @@
     "@actions/exec": "^1.1.1",
     "@actions/github": "^6.0.0",
     "@autoblocks/client": "^0.0.42",
+    "@faker-js/faker": "^8.4.1",
```

Comment:
will drop this, was using to generate outputs

Reply:
Do we still need faker-js here?

Reply:
i was gonna hook it all up in a separate PR but i can just do it here, i'll convert back to draft

Reply:
We can leave it if you want to get this in! Will review as is